docs/Integrating Your HMI/Developing the HMI UI/index.md

```diff
@@ -294,6 +300,18 @@ As an example, if SDL Core requests to change the layout to the `MEDIA` template
               "width":500,
               "rows":1
             },
+            {
+              "name":"subtleAlertText1",
+              "characterSet":"TYPE2SET",
+              "width":500,
+              "rows":1
+            },
+            {
+              "name":"subtleAlertText2",
+              "characterSet":"TYPE2SET",
+              "width":500,
+              "rows":1
+            },
```

Comment:
Should `subtleAlertSoftButtonText` not also be included here?

Reply:
```suggestion
            },
            {
              "name":"subtleAlertSoftButtonText",
              "characterSet":"TYPE2SET",
              "width":50,
              "rows":1
            },
```